assets/src/application/ui/display/DebugInfo.tsx

```diff
@@ -31,18 +32,16 @@ const DebugInfo: React.FC<DebugInfoProps> = ({ data, asJson = true, asCollapse =
 
 	const output = <Pre>{dataToRender}</Pre>;
 
+	// define it here to avoid conditional call of hook
+	const handleToggle = useCallback(() => setShow((v) => !v), [setShow]);
 	if (!asCollapse) {
 		return output;
 	}
 	const buttonText = show ? __('Hide Debug Info') : __('Show Debug Info');
-	const handleToggle = useCallback(() => setShow(!show), [show]);
```

Comment:
This has happened again 😄

Reply:
isn't there an eslint rule for that?

Reply:
> what's happened again?

Conditional calling of React Hook

>isn't there an eslint rule for that?

Yeah, react-hooks-eslint works, but not for us 🤷